corehq/apps/reminders/views.py

```diff
@@ -102,7 +102,7 @@ def page_context(self):
         today = timezone_now.date()
 
         def adjust_next_fire_to_timezone(reminder_utc):
-            return ServerTime(reminder_utc.next_fire).user_time(timezone).done()
+            return ServerTime(reminder_utc.next_fire).user_time(timezone).done().replace(tzinfo=None)
```

Comment:
Ah, nice. Yeah I'm totally comfortable with that then. Wasn't a deal-breaker to begin with, but even less so in code we're actively replacing.